content/docs/alerting/configuration.md

```diff
@@ -520,16 +520,18 @@ fields:
 
 ### `<action_config>`
 
-The fields are documented in the [Slack API documentation](https://api.slack.com/docs/message-attachments#action_fields).
+The fields are documented in the [Slack API documentation for message attachments](https://api.slack.com/docs/message-attachments#action_fields) and the [Slack API documentation for interactive messages](https://api.slack.com/docs/interactive-message-field-guide#action_fields).
 
 ```yaml
 text: <tmpl_string>
 type: <tmpl_string>
-url: <tmpl_string>
-[ confirm: <action_confirm_field_config> ]
+# either url or name and value are mandatory
```

Comment:
added note and mark both

Reply:
The sentence needs to start with a capital E and end with a dot.

Reply:
🙂